qemu/tests/hpt_max_page_size.py

```diff
@@ -26,12 +26,12 @@ def run(test, params, env):
     """
     def _check_meminfo(key):
         meminfo = session.cmd_output("grep %s /proc/meminfo" % key)
-        actual_value = re.search(r'\d+', meminfo)
+        actual_value = re.search(r'\d{4,}', meminfo)
         return actual_value.group(0) if actual_value else ""
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
-    session = vm.wait_for_login()
+    session = vm.wait_for_serial_login(timeout=1800)
```

Comment:
I recommend moving it to the cfg file for easy configuration, and only for TCG part.

Reply:
By the way, how to estimate the correct time out, we had better run this job in a new installed guest and check the time since it will take much more time to boot up the guest with tcg mode for the first time. Thanks.